<launch>

	<arg name="cam" default="camera1" />
	<arg name="dev_id" default="1" /> 
	<arg name="downsample" default="1" />

	<!-- Start the saver node -->
	<node pkg="record_rgbd_service" type="record_rgbd_service" name="$(arg cam)_save_images" output="screen">
   	      <param name="camera_name" value="$(arg cam)" />
	      <param name="device_id" value="$(arg dev_id)" />
	      <param name="downsample" value="$(arg downsample)" />
  	</node>
	
	<!-- Save mapping from camera to dev_video -->
	<node name="camera_mapping_$(arg cam)" 
	      pkg="ar_track_service" 
	      type="save_mapping.py"
	      output="screen"
	      args="--dev_id=$(arg dev_id) --cam=$(arg cam)" />

	<!-- Save type -->
	<node name="camera_type_$(arg cam)" 
	      pkg="ar_track_service" 
	      type="save_type.py"
	      output="screen"
	      args="--type=rgbd --cam=$(arg cam)" />

	
</launch>
